packages/mobile/test/schemas.ts

```diff
@@ -291,6 +291,15 @@ export const v1Schema = {
   },
 }
 
+export const v2Schema = {
```

Comment:
Don't create a new schema version when you're only adding properties. We would only do that when we make backwards incompatible changes